src/appleseed/renderer/kernel/lighting/pathtracer.h

```diff
@@ -355,6 +355,14 @@ size_t PathTracer<PathVisitor, VolumeVisitor, Adjoint>::trace(
             continue;
         }
 
+        // Apply OSL holdout.
```

Comment:
Maybe "Handle" rather than "Apply"?

Reply:
I'm not entirely sure to understand why holdout must be handled both in the Generic Sample Renderer and here.

Reply:
Ok, I checked and yes, even with current version this section is not required, but I think it speeds things up in case of the matte surface.